go.mod

```diff
@@ -3,23 +3,17 @@ module github.com/flatcar-linux/locksmith
 go 1.14
 
 require (
-	github.com/coreos/go-semver v0.3.0 // indirect
 	github.com/coreos/go-systemd v0.0.0-20191104093116-d3cd4ed1dbcf
 	github.com/coreos/pkg v0.0.0-20180928190104-399ea9e2e55f
 	github.com/godbus/dbus v4.1.0+incompatible // indirect
-	github.com/godbus/dbus/v5 v5.0.3
-	github.com/gogo/protobuf v1.3.1 // indirect
+	github.com/godbus/dbus/v5 v5.0.4
 	github.com/hashicorp/errwrap v1.1.0 // indirect
 	github.com/rkt/rkt v1.30.0
-	go.etcd.io/etcd v0.0.0-00010101000000-000000000000
-	go.uber.org/zap v1.16.0 // indirect
-	golang.org/x/net v0.0.0-20201031054903-ff519b6c9102
-	google.golang.org/grpc v1.33.2 // indirect
+	github.com/stretchr/testify v1.7.0
```

Comment:
I wouldn't add a testify as a dependency here. Is it _really_ worth it for few lines saved?